ci/cscs-ci.yml

```diff
@@ -184,6 +189,9 @@ build_py38_image_x86_64:
     # Another problem, observed in test stage, is that gpu tests hang in combination with CUDA MPS,
     # when high test parallelism is used.
     NUM_PROCESSES: 16
+    SLURM_ACCOUNT: g154
+    SLURM_RESERVATION: daint
+    SLURM_TIMELIMIT: '00:15:00'
```

Comment:
`SLURM_TIMELIMIT` is already specified at line 160

Please also add a `TODO` comment to remove these lines once daint reservation is no longer needed.